net.codelogistics.clicker.json

```diff
@@ -0,0 +1,81 @@
+{
+    "id" : "net.codelogistics.clicker",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "46",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "clicker",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "libportal",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Ddocs=false",
+                "-Dintrospection=true"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/flatpak/libportal.git",
+                    "tag": "0.7.1",
+                    "x-checker-data": {
+                        "type": "git",
+                        "tag-pattern": "^([\\d.]+)$"
+                    },
+                    "commit": "e9ed3a50cdde321eaf42361212480a66eb94a57a"
+                }
+            ]
+        },
+        {
+	    "name": "python3-libevdev",
+	    "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"libevdev\" --no-build-isolation"
+	    ],
+	    "sources": [
+		{
+		    "type": "file",
+		    "url": "https://files.pythonhosted.org/packages/b0/49/2fe589ce1fa6ca0f05ae0b1717923650f2cc6eec6307c71fbc7789738902/libevdev-0.11.tar.gz",
+		    "sha256": "e9ca006a4df2488a60bd9a740011ee948d81904be2364f017e560169508f560f",
+		    "x-checker-data": {
+                        "type": "pypi",
+                        "name": "libevdev"
+                    }
+		}
+	    ]
+	    },
+        {
+            "name": "clicker",
+            "builddir": true,
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://codeberg.org/eyekay/clicker.git",
+                    "tag": "0.1.5",
+                    "x-checker-data": {
+                        "type": "git",
+                        "tag-pattern": "^([\\d.]+)$"
+                    },
+                    "commit": "cc4cc727bca6fdbcc58c02e2260e8a5e4de4278f"
```

Comment:
```suggestion
                    "tag": "0.1.5",
                    "commit": "cc4cc727bca6fdbcc58c02e2260e8a5e4de4278f",
                    "x-checker-data": {
                        "type": "git",
                        "tag-pattern": "^([\\d.]+)$"
                    }
```

Reply:
I've already updated this, I committed it while the bot was building the previous commit :facepalm:

Reply:
The x-checker para should be at the end, the bot will reformat it on next update. This avoids that.

Currently you have `commit` as the last property.